apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/NotesForm.tsx

```diff
@@ -0,0 +1,187 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
+import ChatBubbleBottomCenterIcon from "@heroicons/react/24/outline/ChatBubbleBottomCenterIcon";
+import { useMutation } from "urql";
+import isEmpty from "lodash/isEmpty";
+
+import { Button, ToggleSection } from "@gc-digital-talent/ui";
+import { Submit, TextArea } from "@gc-digital-talent/forms";
+import {
+  FragmentType,
+  Maybe,
+  PoolCandidate,
+  getFragment,
+  graphql,
+} from "@gc-digital-talent/graphql";
+import { toast } from "@gc-digital-talent/toast";
+import { commonMessages, formMessages } from "@gc-digital-talent/i18n";
+
+import adminMessages from "~/messages/adminMessages";
+import useToggleSectionInfo from "~/hooks/useToggleSectionInfo";
+import ToggleForm from "~/components/ToggleForm/ToggleForm";
+
+export const NotesForm_Fragment = graphql(/* Graphql */ `
+  fragment NotesForm on PoolCandidate {
+    id
+    notes
+  }
+`);
+type FormValues = {
+  notes?: PoolCandidate["notes"];
+};
+
+interface NotesFormProps {
+  poolCandidate: FragmentType<typeof NotesForm_Fragment>;
+}
+
+const PoolCandidate_UpdateNotesMutation = graphql(/* GraphQL */ `
+  mutation PoolCandidate_UpdateNotes($id: UUID!, $notes: String) {
+    updatePoolCandidateNotes(id: $id, notes: $notes) {
+      id
+      notes
+    }
+  }
+`);
+
+const Display = ({ notes }: { notes?: Maybe<string> }) => {
+  const intl = useIntl();
+
+  return (
+    <>
+      <ToggleForm.FieldDisplay
+        hasError={false}
+        label={intl.formatMessage(adminMessages.notes)}
+      >
+        <div
+          data-h2-margin-top="base(x.5)"
+          data-h2-max-height="base(10rem)"
+          data-h2-overflow="base(scroll)"
```

Comment:
Could be use either the `ScrollArea` or `data-h2-overflow-y="base(auto)"`. This makes it look a little odd :sweat_smile: 

![screenshot_15-May-2024_07-56-1715774196](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/95041766-752e-45c4-a102-c8d6d4446833)

Reply:
Yes that works better, fixed by [3706605](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10354/commits/37066056e191ef5229c800ecde1ba667b11207c7)